README.md

```diff
@@ -329,6 +330,34 @@ Assert.AreEqual(x.Count(), target.Eval("x.Count()"));
 - Generics, only partially supported (only implicit, you cannot invoke an explicit generic method)
 - Params array (see C# `params` keyword)
 
+### Lambda expressions
+Dynamic Expresso has partial supports of lambda expressions. For example, you can use any Linq method:
+
+```csharp
+var x = new string[] { "this", "is", "awesome" };
+var target = new Interpreter()
+	.EnableLambdaExpressions(true) // enable lambda expressions
```

Comment:
We should now use `InterpreterOptions` ... Here and below...

Reply:
Good point!